Add gaplimit option for yield generators.

domob1812 opened the pull request:
Add a new option `-g` (as with `wallet-tool.py`) for the yield generators that allows controlling the gap limit used in the underlying wallet.

AdamISZ commented:
Please open pull requests against the develop branch. Thanks. Looks good (I'm actually a bit bewildered, didn't realise we didn't have that!).

domob1812 commented:
Ah, thanks for the hint!  I've rebased now and changed this PR to be for `develop`.

I've been changing the gaplimit in the version I run for quite some time now (just hardcoded in the code), but I think it makes sense to have a proper option for that. ;)

AdamISZ commented:
Yes it does! Thanks .. :confused: (can't remember if we didn't have it or it was botched up somewhere). I'll take another look in a bit.

AdamISZ commented:
I'm not sure why the travis test didn't trigger, but it probably has something to do with the way you switched from master to develop. Can you re-create it as a new PR based off develop and close this one? I think that should fix it. I think I'll just merge it then.

domob1812 commented:
Done in #613.  Closing this one.
